config/environments/test.rb

```diff
@@ -1,4 +1,16 @@
 Rails.application.configure do
+  {
+    ENGLISH_URI:  '/rest/v1/language/100/',
```

Comment:
Agree with Tuuli; outside of username/password, this is stuff that could go into `production.rb` without a problem.